src/exception/types.ts

```diff
@@ -65,6 +87,8 @@ export const isProgramExceptionRecord = (input: any): input is ProgramExceptionR
   );
 };
 
+export const isEntityExceptionRecord = (input: any): input is any => true;
```

Comment:
this is extra validation

Reply:
leaving as is to not break parseTsv func contract

Reply:
Does this do anything at all? This looks like placeholder code for actual implementation, if so there should at least be some comments explaining its purpose.

This actually seems quite dangerous as it will take any variable and make it `any` type, removing all type checking.